src/@next/components/atoms/RichTextContent/RichTextContent.tsx

```diff
@@ -0,0 +1,14 @@
+import draftToHtml from "draftjs-to-html";
+import React from "react";
+
+import { IProps } from "./types";
+
+export const RichTextContent: React.FC<IProps> = ({ descriptionJson }) => {
+  return descriptionJson ? (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: draftToHtml(JSON.parse(descriptionJson)),
+      }}
+    />
+  ) : null;
```

Comment:
As Jakub suggested lately maybe lests use `return parameter && <some code>` instead of ternary operator. If descriptionJson will be null it will be returned automatically as first argument in a && b expression